### ISSUE TITLE
SVG output contains garbled `xlink:xlink:href` attribute

### ISSUE THREAD

**Issue by vaskevich** [mathjax/MathJax-src#365]
While trying to generate an SVG, with some relatively straightforward code:

```ts
const adaptor = liteAdaptor();
RegisterHTMLHandler(adaptor);

const Tex = new TeX({packages: AllPackages});
const Svg = new SVG({});
const MathJaxDoc = mathjax.document('', {InputJax: Tex, OutputJax: Svg});

const node = MathJaxDoc.convert(source, {display: true});
const svgContent = adaptor.innerHTML(node);
```

However, this is outputting the following broken SVG for a simple "1+2+3" formula:

```svg
<svg style="vertical-align: -0.186ex" 
  xmlns="http://www.w3.org/2000/svg" width="8.925ex" height="1.692ex" role="img" focusable="false" viewBox="0 -666 3944.9 748" 
  xmlns:xlink="http://www.w3.org/1999/xlink">
  <defs>
    <path id="MJX-2-TEX-N-31" d="M213 578L200 573Q186 568 160 563T102 556H83V602H102Q149 604 189 617T245 641T273 663Q275 666 285 666Q294 666 302 660V361L303 61Q310 54 315 52T339 48T401 46H427V0H416Q395 3 257 3Q121 3 100 0H88V46H114Q136 46 152 46T177 47T193 50T201 52T207 57T213 61V578Z"></path>
    <path id="MJX-2-TEX-N-2B" d="M56 237T56 250T70 270H369V420L370 570Q380 583 389 583Q402 583 409 568V270H707Q722 262 722 250T707 230H409V-68Q401 -82 391 -82H389H387Q375 -82 369 -68V230H70Q56 237 56 250Z"></path>
    <path id="MJX-2-TEX-N-32" d="M109 429Q82 429 66 447T50 491Q50 562 103 614T235 666Q326 666 387 610T449 465Q449 422 429 383T381 315T301 241Q265 210 201 149L142 93L218 92Q375 92 385 97Q392 99 409 186V189H449V186Q448 183 436 95T421 3V0H50V19V31Q50 38 56 46T86 81Q115 113 136 137Q145 147 170 174T204 211T233 244T261 278T284 308T305 340T320 369T333 401T340 431T343 464Q343 527 309 573T212 619Q179 619 154 602T119 569T109 550Q109 549 114 549Q132 549 151 535T170 489Q170 464 154 447T109 429Z"></path>
    <path id="MJX-2-TEX-N-33" d="M127 463Q100 463 85 480T69 524Q69 579 117 622T233 665Q268 665 277 664Q351 652 390 611T430 522Q430 470 396 421T302 350L299 348Q299 347 308 345T337 336T375 315Q457 262 457 175Q457 96 395 37T238 -22Q158 -22 100 21T42 130Q42 158 60 175T105 193Q133 193 151 175T169 130Q169 119 166 110T159 94T148 82T136 74T126 70T118 67L114 66Q165 21 238 21Q293 21 321 74Q338 107 338 175V195Q338 290 274 322Q259 328 213 329L171 330L168 332Q166 335 166 348Q166 366 174 366Q202 366 232 371Q266 376 294 413T322 525V533Q322 590 287 612Q265 626 240 626Q208 626 181 615T143 592T132 580H135Q138 579 143 578T153 573T165 566T175 555T183 540T186 520Q186 498 172 481T127 463Z"></path>
  </defs>
  <g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)">
    <g data-mml-node="math">
      <g data-mml-node="mn">
        <use xlink:xlink:href="#MJX-2-TEX-N-31"></use>
      </g>
      <g data-mml-node="mo" transform="translate(722.2, 0)">
        <use xlink:xlink:href="#MJX-2-TEX-N-2B"></use>
      </g>
      <g data-mml-node="mn" transform="translate(1722.4, 0)">
        <use xlink:xlink:href="#MJX-2-TEX-N-32"></use>
      </g>
      <g data-mml-node="mo" transform="translate(2444.7, 0)">
        <use xlink:xlink:href="#MJX-2-TEX-N-2B"></use>
      </g>
      <g data-mml-node="mn" transform="translate(3444.9, 0)">
        <use xlink:xlink:href="#MJX-2-TEX-N-33"></use>
      </g>
    </g>
  </g>
</svg>
```

Specifically, notice the `<use xlink:xlink:href="#MJX-2-TEX-N-33"></use>`. From what I can tell, the issue is that `xlink` is being added by both the SVG wrapper and the LiteAdaptor:

https://github.com/mathjax/MathJax-src/blob/070448bc97b97fe83e9ec8027665373d347f1032/ts/adaptors/liteAdaptor.ts#L426-L428

https://github.com/mathjax/MathJax-src/blob/070448bc97b97fe83e9ec8027665373d347f1032/ts/output/svg/Wrapper.ts#L319

**Comment by dpvc** [comment]
Thanks for the report, and the proposed PR.  The situation is a bit more complicated that it at first appears, and I will make comments on the PR to address those complications.

**Comment by dpvc** [comment]
This is fixed in PR #417.
